ubuntu20/packages.txt

```diff
@@ -1,6 +1,7 @@
 automake
 binutils
 ca-certificates
+cmake
```

Comment:
Ubuntu 20's cmake is too old, we are instead downloading a release in its `Dockerfile`:
```suggestion
```